stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -504,6 +516,42 @@ public struct Unsafe${Mutable}RawBufferPointer
   }
 
   %  if mutable:
+  
+  /// Initializes the memory referenced by this buffer with the given value,
+  /// binds the memory to the value's type, and returns a typed buffer of the
+  /// initialized memory.
+  ///
+  /// The memory referenced by this buffer must be uninitialized or
+  /// initialized to a trivial type, and must be properly aligned for
+  /// accessing `T`.
+  ///
+  /// After calling this method on a raw buffer with non-nil `baseAddress` `b`, 
+  /// the region starting at `b` and continuing up to
+  /// `b + self.count - self.count % MemoryLayout<T>.stride` is bound to type `T` and
+  /// initialized. If `T` is a nontrivial type, you must eventually deinitialize
+  /// or move from the values in this region to avoid leaks. If `baseAddress` is 
```

Comment:
Sorry, I'm confused as to the meaning of "move from the values"--is this a typo? could it be rephrased?

Reply:
move as in move out

Reply:
So, “...move the values out of the this region...”?